app/src/components/governance/PageProposal.vue

```diff
@@ -170,7 +170,11 @@
       <ModalVoteSubstrate
         v-if="currentNetwork.network_type === `polkadot`"
         ref="modalVote"
-        :proposal-id="proposalId"
+        :proposal-id="
+          currentNetwork.network_type === `polkadot`
+            ? proposal.proposalId
+            : proposalId
```

Comment:
```suggestion
        proposal.proposalId
```

Reply:
not needed as you already do  a v-if for the component

Reply:
true